diff --git a/node_modules/@types/prettier/index.d.ts b/node_modules/@types/prettier/index.d.ts
index 6a75a84..3f40256 100755
--- a/node_modules/@types/prettier/index.d.ts
+++ b/node_modules/@types/prettier/index.d.ts
@@ -172,7 +172,7 @@ export interface RequiredOptions extends doc.printer.Options {
     /**
      * Specify plugin directory paths to search for plugins if not installed in the same `node_modules` where prettier is located.
      */
-    pluginSearchDirs: string[];
+    pluginSearchDirs: string[] | false;
     /**
      * How to handle whitespaces in HTML.
      * @default 'css'
