diff --git a/node_modules/type-coverage-core/dist/tsconfig.js b/node_modules/type-coverage-core/dist/tsconfig.js
index d550573..d04f109 100644
--- a/node_modules/type-coverage-core/dist/tsconfig.js
+++ b/node_modules/type-coverage-core/dist/tsconfig.js
@@ -31,15 +31,18 @@ async function getProjectRootNamesAndCompilerOptions(project) {
 }
 exports.getProjectRootNamesAndCompilerOptions = getProjectRootNamesAndCompilerOptions;
 function tryToStatFile(filePath) {
+    const jsonFilePath = filePath.endsWith('.json') ? filePath : filePath + '.json';
     try {
         return {
-            path: filePath,
-            stats: fs.statSync(filePath),
+            path: jsonFilePath,
+            stats: fs.statSync(jsonFilePath),
         };
     }
     catch (_a) {
+        if (jsonFilePath === filePath) {
+            return undefined;
+        }
         try {
-            filePath = filePath + '.json';
             return {
                 path: filePath,
                 stats: fs.statSync(filePath),
